src/containerapp/azext_containerapp/commands.py

```diff
@@ -70,6 +70,19 @@ def load_command_table(self, _):
         g.custom_command('delete', 'delete_managed_environment', supports_no_wait=True, confirmation=True, exception_handler=ex_handler_factory())
         g.custom_command('update', 'update_managed_environment', supports_no_wait=True, exception_handler=ex_handler_factory())
 
+    with self.command_group('containerapp job') as g:
```

Comment:
Is this command group in preview?
Add `is_preview=True `to the command group if all command under the group is preview.
example: https://github.com/Azure/azure-cli-extensions/pull/6292